src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -29,14 +35,19 @@ object SharedFileSystem {
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
 
+  val DockerLocalizers = localizePathAlreadyLocalized _ +: (LocalizationStrategies collect {
+    case "hard-link" => localizePathViaHardLink _
+    case "copy" => localizePathViaCopy _
```

Comment:
`Localizers` explicitly throws `UnsupportedOperationException` on match failure, but this will suffer some sort of match error.